modules/core/src/main/java/org/apache/ignite/internal/processors/management/ManagementConsoleSender.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.management;
+
+/**
+ * Control Center sender.
+ */
+public interface ManagementConsoleSender {
```

Comment:
Could you clarify in the javadoc, what this interface is for. Where is it supposed to be used?

Reply:
Improve javadoc and reuse this interface in all agent processors

Reply:
OK, I see. So, the issue is that we don't have such method in IgniteLogger. I created a ticket for this improvement: https://ggsystems.atlassian.net/browse/GG-28856
Such error handling will lead to a lot of exceptions printed when a node is stopped. As far as I can see, these errors can only occur when a node is stopped, so the reason behind it is almost always in something else. Let's keep it as warning for now.